src/textures/mlx_xpm42.c

```diff
@@ -206,3 +206,28 @@ void mlx_delete_xpm42(xpm_t* xpm)
 	free(xpm->texture.pixels);
 	free(xpm);
 }
+
+char *convert_path(char* path)
+{
+    // Allocate memory for the new string
+    char *new_str = malloc(sizeof(char) * (strlen(path) + 3));
+    if (new_str == NULL) 
+	{
+        printf("Error: malloc failed\n");
```

Comment:
Use `mlx_errno`, don't just print out the error, users should be responsible to print the error when necessary.